apps/uvc/main.py

```diff
@@ -28,8 +31,12 @@
 uvc = pipeline.createUVC()
 cam_rgb.video.link(uvc.input)
 
+
 # Pipeline defined, now the device is connected to
-with dai.Device(pipeline) as device:
+with dai.Device(pipeline, usb2Mode=args.usbSpeed == "usb2") as device:
+    if device.getDeviceInfo().desc.protocol == dai.XLinkProtocol.X_LINK_USB_VSC and device.getUsbSpeed() not in (dai.UsbSpeed.SUPER, dai.UsbSpeed.SUPER_PLUS):
+        print("This app is temporarily disabled with USB2 connection speed due to known issue. We're working on resolving it. In the meantime, please try again with a USB3 cable/port for the device connection")
```

Comment:
Just bringing up:
We could use colored prints for different levels, like in gen1. If there's a library that support windows then that'd great.

Reply:
@szabi-luxonis I changed it so now we send these print statements to `stderr` output - this way we have it cross-platform and visible better, thanks!